src/k8s-engine/minikube.js

```diff
@@ -265,3 +266,28 @@ async function startAgain(obj) {
 function sleep(delay) {
   return new Promise((resolve) => setTimeout(resolve, delay));
 }
+
+function qq(s) {
```

Comment:
See the other PR — JSDoc at least; a better function name would be nice too ([perl operators](https://perldoc.perl.org/perlop#qq/STRING/) isn't the greatest name).  Using a proper shell escaping package would be an option too.